src/test/java/online/rabko/basketball/BasketballApplicationTests.java

```diff
@@ -2,8 +2,10 @@
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
 
 @SpringBootTest
+@ActiveProfiles("test")
```

Comment:
as the number of test classes grows we'll start forgetting about mentioning the profile here

lets update our `build.gradle.kts` to something like

```kotlin
tasks {
    ...
    test {
        systemProperty("spring.profiles.active", "test")
    }
}
```